tests/dagnabbit/dagTestH

```diff
@@ -0,0 +1,40 @@
+<serial>
+
+<parallel>
+
+jobsub_submit --group=uboone --resource-provides=usage_model=DEDICATED,OPPORTUNISTIC --lines='+SingularityImage=\"/cvmfs/singularity.opensciencegrid.org/fermilab/fnal-wn-sl7:latest\"' --expected-lifetime=3h --memory=1000MB --append_condor_requirements='(TARGET.HAS_CVMFS_uboone_opensciencegrid_org==true)' --append_condor_requirements='(TARGET.HAS_CVMFS_uboone_osgstorage_org==true)' --append_condor_requirements='(TARGET.HAS_SINGULARITY=?=true)' file://jobA.sh  --max_files 10  --prestage_fraction 0.750000  --logdir /pnfs/uboone/scratch/users/greenlee/v08_00_00_55/swizzle2/swizzle2
```

Comment:
Do we want to have tests that are tied to Herb's account?  I'm fine keeping them in as a temporary solution to make sure we get good tests, but we should generalize it as soon as possible afterwards.

Reply:
Good catch.  I switched it to $USER